reposerver/repository/repository.proto

```diff
@@ -25,6 +25,7 @@ message ManifestRequest {
     repeated string apiVersions = 15;
     // Request to verify the signature when generating the manifests (only for Git repositories)
     bool verifySignature = 16;
+    string appName = 17;
```

Comment:
I was unsure whether to reuse the field `appLabelValue` here, so I introduced a new field `appName`. But `appLabelValue` will contain the application name as well.

Reply:
I understand the hesitation. It is unclear whether `appLabelValue` must contain app name or has something else. 

I'm pretty sure we always pass application name. Can you remove `appLabelValue` field altogether and use `appName` instead? Repo server API is private so it is ok to make backward incompatible changes.

Reply:
This is actually a great idea, otherwise we'd have an unnecessary redundancy in every API call. Will do the necessary changes!

Reply:
Changed `appLabelValue` to `appName` everywhere, please have another look.